webui/react/src/components/JsonGlossary.tsx

```diff
@@ -0,0 +1,32 @@
+import Glossary from 'hew/Glossary';
+import React from 'react';
+
+import { JsonObject } from 'types';
+import { isObject } from 'utils/data';
+
+type TextTransfomer = (key: string) => string;
+
+interface Props {
+  json: JsonObject;
+  translateLabel?: TextTransfomer;
+}
+
+const JsonGlossary: React.FC<Props> = ({ json, translateLabel }: Props) => {
+  const content = Object.entries(json).map(([label, jsonValue]) => {
+    let textValue = '';
+    if (isObject(jsonValue)) {
+      textValue = JSON.stringify(jsonValue, null, 2);
+    } else if (jsonValue === '' || jsonValue === null || jsonValue === undefined) {
+      textValue = 'N/A';
+    } else {
+      textValue = jsonValue.toString();
+    }
+    return {
+      label: typeof label === 'string' && translateLabel ? translateLabel(label) : label,
```

Comment:
nit: because `isObject` is used in the file, I would suggest import `isString` from there too